src/components/LoanCards/KivaClassicBasicLoanCard.vue

```diff
@@ -3,52 +3,57 @@
 		class="kv-tailwind tw-w-[336px]"
 		:id="`${loanId}-loan-card`"
 	>
-		<!-- Borrower image w/location <summary-tag> -->
+		<!-- Borrower image w/ location <summary-tag> -->
 		<kv-loading-placeholder
 			v-if="isLoading"
 			class="tw-mb-1 tw-rounded" :style="{width: '100%', height: '15.75rem'}"
 		/>
-
 		<div
 			v-if="!isLoading"
 			class="tw-relative"
 		>
-			<borrower-image
-				class="
-				tw-w-full
-				tw-bg-black
-				tw-rounded
-			"
-				:alt="'photo of ' + borrowerName"
-				:aspect-ratio="3 / 4"
-				:default-image="{ width: 336 }"
-				:hash="imageHash"
-				:images="[
-					{ width: 336, viewSize: 1024 },
-					{ width: 336, viewSize: 768 },
-					{ width: 416, viewSize: 480 },
-					{ width: 374, viewSize: 414 },
-					{ width: 335, viewSize: 375 },
-					{ width: 280 },
-				]"
-				:city="city"
-				:state="state"
-				:country-name="countryName"
-				:distribution-model="distributionModel"
-			/>
-			<summary-tag
-				v-if="countryName"
-				class="tw-absolute tw-bottom-2 tw-left-1"
-				:city="city"
-				:state="state"
-				:country-name="countryName"
+			<!-- If allSharesReserved, disable link by making it a span -->
+			<router-link
```

Comment:
Does this work? This is interesting syntax. I would expect this to be `<component :is="allSharesReserved ? 'span' : 'router-link'"`

Reply:
What I have in place does work, but I don't see a difference between what I have in there and what you suggested above. Am I misunderstanding you?

Reply:
This is rendering in storybook. 
**When router-link renders as a link (while loan is fundraising)**
<img width="610" alt="Screen Shot 2021-09-24 at 1 54 31 PM" src="https://user-images.githubusercontent.com/1521381/134732742-4206b033-a1a6-4a68-b3ba-7f2e6ccb8b41.png">

**When router-link renders as a span (while allSharesReserved)**
<img width="591" alt="Screen Shot 2021-09-24 at 1 55 18 PM" src="https://user-images.githubusercontent.com/1521381/134732841-f1aeae45-2f97-420c-be59-aabf53886c29.png">

Reply:
Is the `span` still clickable? Could you also empty the 'to' field in this case?

Reply:
I'll be sure to verify this functionality in dev across the major browsers.

Reply:
<router-link is used in the same way within [CheckoutItemImg.vue on lines 2 & 3](https://github.com/kiva/ui/blob/master/src/components/Checkout/CheckoutItemImg.vue#L3). That change was introduced about 8 months ago.

Reply:
@mcstover I had that :to field return an empty string at an earlier point in my code review, but it was raised that it was a little redundant to have the allSharesReserved check on both fields, so I removed that. The Span is not clickable when returned and the mouse over event is that of a span. Is it prefered to clear out to :to field in the allSharesReserved state?

Reply:
SOunds good, thanks for checking! It's fine as is.

Reply:
> What I have in place does work, but I don't see a difference between what I have in there and what you suggested above. Am I misunderstanding you?
> 
> `<router-link :is="allSharesReserved ? 'span' : 'router-link'"`

@BoulderBrains I meant to literally make it: 
`<component
:is="allSharesReserved ? 'span' : 'router-link'"
:to="`/lend/${loanId}`"
v-kv-track-event="['Lending', 'click-Read more', 'Photo', loanId]"
>`

But looks like you tested it and it's working, so it's fine as is. :thumbs